### PR TITLE
Bump checkout and upload-artifacts

### PR DESCRIPTION
**Pull request by maxg-square**
Update to current version of checkout and upload-artifacts

### CODE REVIEW

**Review by kyleve — APPROVED** [review]
LGTM if it builds!

**Review by robmaceachern — APPROVED** [review]
🤞🏽

File: BlueprintUICommonControls/Tests/Sources/AccessibilityContainerTests.swift [hunk 1]
```diff
@@ -6,7 +6,7 @@ import XCTest
 class AccessibilityContainerTests: XCTestCase {
 
     func test_recursiveAccessibleSubviewsIncludesContainedElements() {
-
+        XCTFail("Test failure")
```
Comment (watt) [inline BlueprintUICommonControls/Tests/Sources/AccessibilityContainerTests.swift:9]: I think you'll need to fail a snapshot test to get any test artifacts. This one's just a unit test.

**Review by watt — APPROVED** [review]

File: BlueprintUICommonControls/Tests/Sources/AccessibilityContainerTests.swift (hunk 1 above)
Comment (maxg-square) [reply BlueprintUICommonControls/Tests/Sources/AccessibilityContainerTests.swift:9]: Yeah I was noticing it wasn't dirtying that screenshots directly, figured that might be why 👍

**Comment by maxg-square** [comment]
Verified artifacts upload okay post-change.

**Pull request merged**